homeworks/yehor.honcharenko_NOcode-developer/GamesProject/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Game Project</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header>
+            <nav class="header">
+                <!-- <ul>
+                    <li><a href="#">Games</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Login</a></li>
+                    </ul> -->
+            </nav>
+        </header>
+        <main>
+            <article class="about-me">
+                <section class="img-about">
+                    <img src="./assets/img/image-about1.svg" alt="image about us">
+                </section>
+                <section class="text-about">
+                    <h1>About me</h1>
+                    <p>
+                        Hi! My name is [Text] and I'm a Junior Frontend
+                        Developer. I am already familiar with main Web
+                        Technologies like HTML, CSS, JavaScript and Git version
+                        control system.
+                    </p>
+                    <p>
+                        This page was developed during the course
+                        <a href="https://www.mastersacademy.education/frontend-for-beginners-it"
+                            target="_blank">'Frontend for beginners' from Masters Academy in
+                            2023</a>.
+                    </p>
+                    <p>
+                        This is a social project from MOCG company where I got
+                        an opportunity to work with Frontend mentors and to
+                        create my own small project for the portfolio.
+                    </p>
+                    <p>
+                        You can contact me via [Links to Socials Nets] and/or
```

Comment:
Same here: you can add links to your Social nets and GitHub now. Or later, on the final stage of the project.

Reply:
I thought it was just plain text; it was my inattention. I've fixed it and added the links